src/components/variant-select.js

```diff
@@ -5,8 +5,8 @@ import loadWebFonts from "../utilities/load-web-fonts"
 
 const VariantSelect = ({ value, options, onSelect, familyName }) => {
   useEffect(() => {
-    loadWebFonts([`${familyName}:${value}`])
-  }, [value])
+    loadWebFonts([`${familyName ? familyName : "Roboto"}:${value}`])
```

Comment:
TBH This change probably can be ignored, as I made the `familyName` PropType required.